Dockerfile

```diff
@@ -1,3 +1,5 @@
-FROM alpine:3.18
+FROM mcr.microsoft.com/cbl-mariner/base/core:2.0
+RUN tdnf install -y azure-cli jq && tdnf clean all
+COPY rg-cleanup.sh /usr/local/bin
 COPY bin/rg-cleanup /usr/local/bin
-ENTRYPOINT [ "rg-cleanup" ]
+ENTRYPOINT [ "rg-cleanup.sh" ]
```

Comment:
Will this break the existing binary? Is the plan to run the container twice? once with the go binary and once with the script?

Reply:
this new script invokes the existing rg-cleanup binary in L16

Reply:
doh, I missed that, thanks!